### ISSUE TITLE
KDB Canopy Data : 9795a5d5-06c5-4a35-ac0b-535533d50748

### ISSUE THREAD

**Issue by kyletaylored** [OpenDenton/City-of-Denton-Datasets#138]
**Title:** KDB Canopy Data
**ID:** 9795a5d5-06c5-4a35-ac0b-535533d50748
**Description:** The City of Denton created Urban Tree Canopy to manage, protect and expand Denton's tree canopy cover. Results of the land cover mapping task show that the City of Denton averages 30% tree canopy cover (22,540 acres out of 74,492), 45% was grass and open space (with 33,321 acres), and 14% was impervious (with 10,506 acres)...
**Created:** 01 / 10 / 2017
**Updated:** 01 / 10 / 2017
**State:** active
**Organization:** City of Denton
**URL:** [http://data.cityofdenton.com/dataset/kdb-canopy-data](http://data.cityofdenton.com/dataset/kdb-canopy-data)
**Tags:** Bare Soil, Impervious Surfaces, Land Cover, Non-Canopy Vegetation, Soil, Total Acres, Tree Canopy, Urban Tree Canopy, Vegetation, Water Bodies

### Resources

- [City of Denton Tree Canopy Data](https://pg-cloud.com/DentonTX/),

**Comment by DavidBrunow** [comment]
Unless I am missing something, this data is 0% machine readable.

**Comment by pmmcleod** [comment]
Agree with Dave. Appears to be an interactive UI, not a data feed.

**Comment by pmmcleod** [comment]
This data source points to an embedded browser viewer without an option for downloading tabular data. If tabular data is available, this would be usable data.

**Comment by newswim** [comment]
I typed `data` into the console and got [this big json dump](https://gist.github.com/newswim/14c5e24d44f6a30cbce0af156d348d14) (you'll want to install a [JSON parser](https://chrome.google.com/webstore/detail/jsonview/chklaanhfefbnpoihckbnefhakgolnmc) as well).

I'm still looking it over but the `ecobens ` section looks interesting, also all of the admin settings and lookup tables are available.

**Comment by DavidBrunow** [comment]
I agree it is interesting but since we don't have any guarantee that the format won't change then I don't think it is useful in its current state.

**Comment by newswim** [comment]
Agreed, after looking it over that json file, there's not a lot of interest.

Just to jot down roughly the types of data this (currently inaccessible) set is concerned with:
- PPA (possible planting area) percentage per district
- Urban Tree Canopy per district

**Comment by Texrat** [comment]
regarding datasets: last year some of us put together a hackpad with numerous relevant resources, mostly around atmosphere: https://opendenton.hackpad.com/Open-Denton-Atmospheric-Monitoring-Project#Open-Denton-Atmospheric-Monitoring-Project

**Comment by newswim** [comment]
I forgot about this! There's a lot of general notes:

https://opendenton.hackpad.com
